arc_design_contest/2019/HUST_electrical_load_identification/makefile

```diff
@@ -0,0 +1,47 @@
+# Application name
+APPL ?= load_identification_app
+
+TOOLCHAIN = gnu
+
+BOARD ?= iotdk
+
+EXT_DEV_LIST += adc/ad7991
+
+CUR_CORE = arcem9d
+
+APPL_DEFINES = -DUSE_APPL_MEM_CONFIG -DV2DSP_XY -DMODEL_BIT_DEPTH=16 -g
+
+# optimization Level
+OLEVEL = O0
+
+#
+# root dir of embARC
+#
+EMBARC_ROOT = ../../
```

Comment:
project folder is embarc_osp\embarc_applications\arc_design_contest\2019\HUST_electrical_load_identification.
so the EMBARC_ROOT should be ../../../..

Reply:
Modified